sandbox-tools/merge-repos/src/mergeStagingToMain.ts

```diff
@@ -0,0 +1,996 @@
+
+    return mergeCommitMessage;
+}
+
+export async function movePackage(stagingDetails: IStagingRepoDetails, srcFolder: string, destFolder: string) {
+
+    let theLocalDestPath = path.resolve(path.join(stagingDetails.path, destFolder)).replace(/\\/g, "/") + "/";
+    let theGitDestFolder = destFolder;
+
+    if (srcFolder.length === 0) {
+        // Don't log this if we are in recursively moving
+        log(`Moving Package to ${theGitDestFolder}; Local dest path: ${theLocalDestPath}`);
+    }
+
+    const files = fs.readdirSync(stagingDetails.path + "/" + srcFolder);
+    log(`${files.length} file(s) found in ${stagingDetails.path + "/" + srcFolder} to move`);
+    if (!fs.existsSync(theLocalDestPath)) {
+        fs.mkdirSync(theLocalDestPath, { recursive: true });
+    }
+
+    if (files.length > 0) {
+        let commitMessage = stagingDetails.commitDetails.message;
+
+        commitMessage += `\n### Moving files from ${srcFolder ? srcFolder : "./"} to ${theGitDestFolder}`
+        commitMessage = await moveFolder(stagingDetails.git, commitMessage, stagingDetails.path, srcFolder, theGitDestFolder, 0);
+
+        stagingDetails.commitDetails.message = commitMessage;
+    } else {
+        log(` - No files found in ${stagingDetails.path + "/" + srcFolder}`);
+    }
+}
+
+//---------------------------------------------------------------------------------------------------------------
+// Main Script execution
+//---------------------------------------------------------------------------------------------------------------
+
+if (!_gitRoot) {
+    console.error("Unable to locate the repo root");
+    terminate(2);
+}
+
+addCleanupCallback(async (git: SimpleGit)  => {
+    //await removeTemporaryRemotes(git, reposToSyncAndMerge);
+});
+
+_theArgs = parseArgs({
+    switches: {
+        cloneTo: true,
+        stagingBranch: true,
+        destBranch: true,
+        originRepo: true,
+        test: false,
+        noPr: false,
+        originUser: true,
+        destUser: true
+    },
+    defaults: {
+        values: _theArgs.values,
+        switches: _theArgs.switches
+    }
+});
+
+if (_theArgs.switches.showHelp) {
+    showHelp();
+}
+
+if (_theArgs.failed) {
+    fail(null, `Failed parsing arguments - ${JSON.stringify(_theArgs.errors, null, 4)}`);
+}
+
+const localGit = createGit(_gitRoot, "local.git");
+log(`CWD: ${process.cwd()}; gitRoot: ${_gitRoot}`);
+
+localGit.checkIsRepo().then(async (isRepo) => {
+    if (isRepo) {
+        // Lets go and process the repos
+        log("We have a repo");
+        const originRepo = _theArgs.switches.originRepo;
+        const originRepoUrl = "https://github.com/" + originRepo;
+        const stagingBranch = _theArgs.switches.stagingBranch;
+        const stagingStartPoint = _theArgs.switches.stagingStartPoint;
+        const destBranch = _theArgs.switches.destBranch;
+        let createPr = !_theArgs.switches.noPr;
+        if (_theArgs.switches.test ) {
+            //createPr = false;
```

Comment:
This is the other "final" change (after local test validation) that I'll uncomment (Along with the comment on line 106). This causes the presence of the "-test" switch to also disable the creation of a PR.